### ISSUE TITLE
undefined symbol with dballe 7.35

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/provami#30]
```
$ provami-qt 
provami-qt: symbol lookup error: provami-qt: undefined symbol: _ZNK6dballe2db7Summary8supportsERKNS_5QueryE
```
rebuilding the package results in several errors beginning with:
```
/root/rpmbuild/BUILD/provami-1.4-2/provami/model.cpp: In member function 'void provami::Model::refresh_summary(bool)':
/root/rpmbuild/BUILD/provami-1.4-2/provami/model.cpp:215:18: error: 'Support' is not a member of 'dballe::db::summary'
     db::summary::Support supported = UNSUPPORTED;
                  ^~~~~~~
```
Full travis logs available here:
https://travis-ci.org/ARPA-SIMC/provami/builds/369022342

Internal server `ventiquattro/ventotto/sette` are available, a recent build dir is in
`makerpm@ventiquattro:/homelocal/makerpm/rpmbuild/BUILD/provami-1.4-2`

(I'm aware of the python branch but to my knowledge it hasn't been tested from the users so we still need the current master working)

**Comment by spanezz** [comment]
In branch `dballe` I pushed a version of provami/Qt that builds with the `issue128` branch of DB-All.e.

Since I am not anymore able to deal with the Qt codebase, and I started `dballe-web` as a replacement, I started sacrificing those features in provami that I am not anymore able to maintain. provami is in maintenance-only mode, I'll try to keep it building, but I won't do any more development on it than that.

It should still be useful for the general use cases, but feature-related requests, from now on, belong to dballe-web, not to provami.
